sdk/test-utils/recorder/MIGRATION.md

```diff
@@ -255,6 +261,11 @@ module.exports = function (config) {
 The following configuration options in `karma.config.js` should be **removed**:
 
 ```ts
+// files section
+.concat(isPlaybackMode() || isSoftRecordMode() ? ["recordings/browsers/**/*.json"] : [])
+
+/* ... */
+
```

Comment:
Should we mention the case of Identity? after the improved pipeline policies changes are merged?

Reply:
I am expecting that PR to be merged soon, won't mention that here.

Reply:
ok ok